vllm/engine/llm_engine.py

```diff
@@ -896,6 +902,45 @@ def _process_model_outputs(self,
         request_id: If provided, then only this request is going to be processed
 
         """
+
+        def prefill_update_num_computed_tokens_from_outputs(
```

Comment:
I can't get the point from this function name...also seems like we only use `len(outputs)` instead of the actual outputs. Can we have a better naming, and pass `num_outputs` to be clear?

Reply:
renamed, fixed the arguments and added a docstring.